openbb_terminal/config_terminal.py

```diff
@@ -33,6 +33,27 @@ def change_logging_suppress(new_value: bool):
     set_current_system(updated_system)
 
 
+def change_test_mode(new_value: bool):
```

Comment:
Here wdyt having variable names as arguments instead and have it on `openbb_terminal/core/session/current_system.py`? Think it might scale better as we add new variables to the dataclass.

`set_system_variable("TEST_MODE", True)`


```python3
def set_system_variable(
    name: str,
    value: Any,
):
    """Set system variable

    Parameters
    ----------
    name : str
        Variable name
    value : Any
        Variable value
    """
    current_system = get_current_system()
    updated_system = dataclasses.replace(current_system, **{name: value})  # type: ignore
    set_current_system(updated_system)
```

Reply:
This is very good suggestion, thank you. Just addressed it on the most recent commits!